### DISCUSSION TITLE
Tips for model training with snow

### DISCUSSION THREAD

**Discussion by erasmus83** [blakeblackshear/frigate#15064] [Ask A Question]
The first snowfall in the UK has played havoc with my cameras. The main issue has been falling snow being tracked as either a person or cat.

I’m planning to use some images with snow on the ground for training, with some examples of genuine people and cats in the snow.

Some of the images have the car covered in snow in them which is barely identifiable as a car. Should I be tagging it, or leave it out? 

Any other tips for training / tweaking my config to reduce false alerts?

**Comment by NickM-27** [comment]
If you can see it's a car then I'd that it
